build.psm1

```diff
@@ -914,9 +964,28 @@ Make sure that the proper git submodules are installed by running:
         $outputBufferFilePath = [System.IO.Path]::GetTempFileName()
     }
 
-    $Command += "Invoke-Pester "
+    $testPaths = @{}
+    if ($UseRunspacePool)
```

Comment:
We need to figure out how to make TestHooks session local v/s global as they are currently, before this gets merged.

Reply:
One option, that I don't particularly like, is to put all the test hook tests in the same folder so they are all run serially